assets/app/actions/createDraftBranchName.js

```diff
@@ -0,0 +1,5 @@
+import { encodeB64 } from '../util/encoding';
+
+export default path => {
```

Comment:
ha, yes, I thought you had already seen that! I also didn't realize we left the duplicate code in that file...